### PR TITLE
Close modal buttons focus outline + keyboard accessibility fix

### PR DESCRIPTION
**Pull request by yil039**
# Details

### Summary

* The Esc button 'X' now have grey background focus for the following modals

* Add a class
screen shot 2017-11-09 at 2 57 42 pm

* Import - choose the source
screen shot 2017-11-17 at 6 00 01 pm

* Add a user
screen shot 2017-11-17 at 6 01 22 pm

* Welcome to Kolibri
the ESC button does have a focus but as soon as the user uses TAB to navigate onto
the 'X' the welcome modal closes itself immediately

* Now after the onboarding process, the user is able to close the welcome modal using
both the ENTER key and the ESC key

### Reviewer guidance

For all the modals checked above except the welcome one, just open the corresponding modal
and use TAB key to verify whether the 'X' is in focus, and for the welcome-modal, need to delete
the hidden '.kolibri' folder under the user's home directory and go through a complete new on-
boarding process in order for the welcome-modal to show up, and then follow the same checking
procedure.

### References

* Fixed issue #2380 and #2269 

# Contributor Checklist

- [x] PR has the correct target milestone
- [x] PR has the appropriate labels
- [x] Changes in the PR do not introduce accessibility regressions ([confimed by testing with one of the recommended tools](http://kolibri.readthedocs.io/en/develop/dev/manual_testing.html#accessibility-a11y-testing)) 
- ~Documentation is updated as necessary~
- ~External dependency files are updated (`yarn` and `pip`)~
- ~If internal dependency is updated, link to diff is included~
- [x] Screenshots of any front-end changes are in the PR description
- ~CHANGELOG.rst is updated for high-level changes~
- [x] You've added yourself to AUTHORS.rst if you're not there

# Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] PR has been fully tested manually

### CODE REVIEW

File: kolibri/plugins/management/assets/src/device_management/views/welcome-modal.vue [hunk 1]
```diff
@@ -42,6 +43,9 @@
       welcomeButtonDismissText: 'OK',
     },
     components: { coreModal, kButton },
+    created: function() {
+      window.addEventListener('keyup', this.emitCloseModal);
+    },
```
Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/device_management/views/welcome-modal.vue:48]: This event listener will cause _any_ keypress to close the window, which is not desirable behavior.

File: kolibri/plugins/management/assets/src/device_management/views/welcome-modal.vue [hunk 2]
```diff
@@ -4,6 +4,7 @@
     :title="$tr('welcomeModalHeader')"
     :enableBgClickCancel="false"
     @cancel="emitCloseModal"
+    @keyup.enter="emitCloseModal"
```
Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/device_management/views/welcome-modal.vue:7]: The `core-modal` plugin has a [built-in way](https://github.com/learningequality/kolibri/pull/2637/commits/0d3c8443737e3b43074a554ced22cd21991d2d24) of supporting this behavior.

File: kolibri/core/assets/src/views/core-modal/index.vue [hunk 3]
```diff
@@ -212,7 +212,9 @@
     color: $core-text-default
     border: none
     position: absolute
-
+    &:focus
+      background-color: $core-grey-300
```
Comment (indirectlylit) [inline kolibri/core/assets/src/views/core-modal/index.vue:216]: looks good! I'm going to take this change and apply it more globally, to all `button` instances.

**Comment by indirectlylit** [comment]
> image

In the future, please create independent commits for logically independent changes. Combining them like this makes it difficult to manage changes.

**Comment by indirectlylit** [comment]
migrated your changes to #2637

File: kolibri/plugins/management/assets/src/device_management/views/welcome-modal.vue (hunk 2 above)
Comment (yixuan-liu) [reply kolibri/plugins/management/assets/src/device_management/views/welcome-modal.vue:7]: Oh I see. This fix is much simpler. Thanks Devon!

**Pull request closed without merging**